### PR TITLE
fix #56538 tdn.com

### LINKED ISSUE #56538: tdn.com
### Issue URL (Incorrect Blocking)
 [https://tdn.com/lifestyles/a-place-called-crystal-pool/article_1cc1c521-4c72-505a-a2e4-ce623951eef4.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftdn.com%2Flifestyles%2Fa-place-called-crystal-pool%2Farticle_1cc1c521-4c72-505a-a2e4-ce623951eef4.html)
### Comment
Username: @DenaliBeast
> website doesn't allow blocker to work correctly

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.4.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard French, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#56538

### CODE REVIEW

**Review by zloyden — COMMENTED** [review]
What about content blockers?

**Comment by Sergey-Lyapin** [comment]
> What about content blockers?

There is no detection on my device with the content blocker (wigh US/canadian IP)

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -90,6 +90,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !
 youtube.com##.ytd-single-option-survey-option-renderer
+tdn.com##.frel_overlay
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:93]: I don't see this popup. Please show it.

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt:93]: > > What about content blockers?
> 
> There is no detection on my device with the content blocker (with US/canadian IP)

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt:93]: I can't reproduce it anymore

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**